MathLib/LinAlg/UnifiedMatrixSetters.h

```diff
@@ -13,45 +13,17 @@
 #define MATHLIB_UNIFIED_MATRIX_SETTERS_H
 
 #include <initializer_list>
+#include <Eigen/Eigen>
```

Comment:
I think it would be enough to include <Eigen/Core>. Telling from the <Eigen/Eigen> header on my system it also includes linear solvers and other things.